tools/replay/parse_dump_resources_cli.cpp

```diff
@@ -78,51 +79,36 @@ static bool CheckIndicesForErrors(const gfxrecon::decode::VulkanReplayOptions& v
     // Indices should be provided sorted
     if (vulkan_replay_options.Draw_Indices.size())
     {
-        bool is_2d = false;
         for (const auto& indices : vulkan_replay_options.Draw_Indices)
         {
-            is_2d |= (indices.size() > 0);
-
             if (!AreIndicesSorted(indices))
             {
-                GFXRECON_LOG_ERROR("ERROR - incomplete --dump-resources parameters");
-                GFXRECON_LOG_ERROR("DrawCall indices are not sorted")
+                GFXRECON_LOG_ERROR("ERROR - incorrect --dump-resources parameters");
+                GFXRECON_LOG_ERROR("Draw indices are not sorted")
                 return true;
             }
         }
-
-        if (!is_2d)
```

Comment:
The dimensions check is completely gone now

Reply:
But we check for indices1.size() > 0, which is checking to make sure it is 2d.

Reply:
> But we check for indices1.size() > 0

Can't see it. On which line?

Reply:
Ahh, I was looking at a few lines down where we check RenderPassIndices at line 100.

But Draw_Indices will not be 2d if all dump resources are on TraceRays and Dispatch calls. So I removed the check.